src/commands/server/deploy.ts

```diff
@@ -423,6 +428,37 @@ export default class Deploy extends Command {
   }
 }
 
+function ensureOIDCProviderInstalled(flags: any): Listr.ListrTask {
+  return {
+    title: 'Check if OIDC Provider installed',
+    enabled: () => !flags['skip-oidc-provider-check'] && isKubernetesPlatformFamily(flags.platform),
+    skip: () => {
+      if (flags.platform === 'minikube') {
+        return 'Dex will be automatically installed'
+      }
+    },
+    task: async (_ctx: any, task: any) => {
+      const kube = new KubeHelper(flags)
+      const apiServerPods = await kube.getPodListByLabel('kube-system', 'component=kube-apiserver')
```

Comment:
checking apiserver container arguments looks a bit hacky. Have you checked that there isn't any API to get oidc url or something like that? I really don't know how this is realiable on various kubernetes flavors. (doing quick search I can't find anything so this may be our best bet)

Reply:
I searched it second time and still found nothing. Looks like it is not possible to get via K8S API, API server configuration